package.json

```diff
@@ -31,8 +35,12 @@
     "email": "support@digitalbazaar.com"
   },
   "license": "BSD-3-Clause",
-  "main": "lib/index.js",
-  "dependencies": {},
+  "main": "src/index.js",
+  "dependencies": {
+    "crypto-ld": "digitalbazaar/crypto-ld#auto-set-id",
+    "did-context": "^2.0.0",
+    "x25519-key-pair": "digitalbazaar/x25519-key-pair#auto-set-id"
```

Comment:
did-io would ideally not have any of these dependencies. The README can refer to them to show how to install them if you want them, but the point of the design is to enable users to install what they want to support and nothing more.

Reply:
@dlongley I was just thinking the same thing. In fact, I'd really like to move the `DidDocument` class to its own repo (so that the key-generation dependencies can move with it, and `did-io` itself is very minimalist).